src/integrations/prefect-shell/README.md

```diff
@@ -0,0 +1,174 @@
+# Integrating shell commands into your dataflow with `prefect-shell`
+
+<p align="center">
+    <img src="https://user-images.githubusercontent.com/15331990/216169092-20cc6e77-ee3b-4aef-a8e7-02747eb5a549.png">
+    <br>
```

Comment:
```suggestion
```

Reply:
removed in [28a5a46](https://github.com/PrefectHQ/prefect/pull/12843/commits/28a5a4653cd4aa50d271960968dfc9d56b6c6b31)